src/datomic_toolbox/core.clj

```diff
@@ -21,97 +20,28 @@
 
 (defn db [] (d/db (connection)))
 
+(defn tempid
+  ([]  (d/tempid (partition)))
+  ([n] (d/tempid (partition) n)))
+
 (defn transact [tx-data]
```

Comment:
I feel very uneasy about having multiple arities of the same function scattered across namespaces.